modules/caddyhttp/fileserver/browsetpl.go

```diff
@@ -358,6 +358,9 @@ footer {
 								{{- end}}
 								<span class="name">{{html .Name}}</span>
 							</a>
+							{{- if .IsDir}}
```

Comment:
I am unsure if this should be in the default template?

Maybe see if anyone has an opinion @whitestrake @mholt @elcore

Reply:
Yeah, good question... I'm not sure yet either. Seems a bit niche.

Reply:
Let me know where to put it

Reply:
Also, this has to be thought because it is quite ugly the way i put, maybe a new <td> with an svg icon?